packages/core/src/decorators/__tests__/media-type-examples-override.test.ts

```diff
@@ -0,0 +1,555 @@
+        requestBodies:
+          testRequest:
+            content:
+              application/json:
+                examples:
+                  def:
+                    value:
+                      a: test123
+        responses:
+          okay200:
+            description: json
+            content:
+              application/json:
+                examples:
+                  def:
+                    value:
+                      a: t
+                      b: 3
+
+    `);
+  });
+
+  it('should override examples with ref', async () => {
+    const testDocument = parseYamlToDocument(
+      outdent`
+            openapi: 3.0.0
+            paths:
+              /pet:
+                get:
+                  operationId: getUserById
+                  responses:
+                    '400':
+                      description: bad request
+                      content:
+                        application/json:
+                          examples:
+                            $ref: '#/components/examples/testExample'
+                  requestBody: 
+                    content:
+                      application/json:
+                        examples:
+                          obj:
+                            $ref: '#/components/examples/testExample'
+            components:
+              examples:
+                testExample:
+                  value:
+                    test: 1                          
+    `
+    );
+
+    const { bundle: res } = await bundleDocument({
+      document: testDocument,
+      externalRefResolver: new BaseResolver(),
+      config: await makeConfig(
+        {},
+        {
+          'media-type-examples-override': {
+            operationIds: {
+              getUserById: {
+                request: {
+                  'application/json':
+                    'packages/core/src/decorators/__tests__/resources/request.yaml',
+                },
+                responses: {
+                  '400': 'packages/core/src/decorators/__tests__/resources/response.yaml',
+                },
+              },
+            },
+          },
+        }
+      ),
+    });
+
+    expect(res.parsed).toMatchInlineSnapshot(`
+      openapi: 3.0.0
+      paths:
+        /pet:
+          get:
+            operationId: getUserById
+            responses:
+              '400':
+                description: bad request
+                content:
+                  application/json:
+                    examples:
+                      def:
+                        value:
+                          name: test response name
+            requestBody:
+              content:
+                application/json:
+                  examples:
+                    def:
+                      value:
+                        b: from external file
+      components:
+        examples:
+          testExample:
+            value:
+              test: 1
+
+    `);
+  });
+
+  it('should not override the examples', async () => {
```

Comment:
This is the wrong behavior. It should create the examples.

Reply:
How is the config currently?

Reply:
@adamaltman 

Now it looks like this: 
```yaml
decorators:
  media-type-examples-override:
    operationIds:
       PostPets:
          request: 
              application/json: ./pet-examples.yaml
          responses: 
            '200': ./pet-examples.yaml
            '400': ./pet-errors-examples.yaml
```

Reply:
Yes, you're right.

Reply:
Changed ✅